Rakefile

```diff
@@ -84,4 +90,8 @@ task :validate_sidebar_tree => [:build] do
   puts "Sidebar links validated successfully"
 end
 
-task :validate => [:validate_kit_versions, :validate_links, :validate_sidebar_tree]
+task :validate_spellings => [:build_spellings] do
+  sh('npm run spell-check')
+end
+
+task :validate => [:validate_kit_versions, :validate_links, :validate_sidebar_tree, :validate_spellings]
```

Comment:
Thought for another time: I wonder if CI should check each of these independently? (Currently I suspect a failure in one will mask a failure in another).